src/vsr/replica.zig

```diff
@@ -6405,17 +6411,19 @@ pub fn ReplicaType(
                         self.view_headers.array.get(0).op,
                     },
                 );
-
-                self.send_header_to_replica(
-                    self.primary_index(self.view),
-                    @bitCast(Header.RequestStartView{
-                        .command = .request_start_view,
-                        .cluster = self.cluster,
-                        .replica = self.replica,
-                        .view = self.view,
-                        .nonce = self.nonce,
-                    }),
-                );
+                self.repair_messages_budget -|= 1;
+                if (self.repair_messages_budget > 0) {
```

Comment:
So if `repair_messages_budget` was originally 1, then we reduce it to `0` without sending anything?
That is somewhat surprising, but seems fine. Though it means we should comptime-assert that `vsr_repair_message_budget_max > 1`.

Reply:
Yeah, good point! Set the budget t one max + 1 with a comment explaining why.